index.html

```diff
@@ -34,6 +38,141 @@ <h3>I'm a Software Engineer</h3>
+        </p>
+        <ul>
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </article>
+      <article class="cards professional-art-1">
+        <h2>Professional Art <br> Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </article>
+      <article class="cards professional-art-2">
+        <h2>Professional Art <br> Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </article>
+      <article class="cards professional-art-3">
+        <h2>Professional Art <br> Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </article>
+      <article class="cards professional-art-4">
+        <h2>Professional Art <br> Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </article>
+      <article class="cards professional-art-5">
+        <h2>Professional Art <br> Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </article>
+      <article class="cards professional-art-6">
+        <h2>Professional Art <br> Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </article>
+    </section>
+     <section class="about-me">
+      <h1>About me</h1>
```

Comment:
As stated early kindly make sure you don't have more than one h1 tag on a page 👍.